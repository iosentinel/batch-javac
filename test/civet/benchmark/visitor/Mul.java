package benchmark.visitor;

public class Mul extends BinOp {
	int apply(int x, int y) {
		return x * y;
	}
}
