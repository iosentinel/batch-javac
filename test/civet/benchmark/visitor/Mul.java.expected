package benchmark.visitor;

public class Mul extends BinOp {
    
    public Mul() {
        super();
    }
    
    int apply(int x, int y) {
        return x * y;
    }
    
    static int apply$1010(int x, int y) {
        return x * y;
    }
}